arraykit.c

```diff
@@ -288,6 +289,22 @@ ArrayGO_extend(ArrayGOObject *self, PyObject *values)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+ArrayGO_getnewargs(ArrayGOObject *self, PyObject *Py_UNUSED(unused))
+{
+    if (self->list && update_array_cache(self)) {
+        return NULL;
+    }
+    PyObject *args = PyTuple_New(1);
+    if (!args) {
+        return NULL;
+    }
+    PyTuple_SET_ITEM(args, 0, self->array);
+    Py_INCREF(args);
```

Comment:
`PyTuple_New` gave you a tuple with a reference count of 1. No need to increment it to 2 here, since your caller will hold the only reference after returning:

```suggestion
```

Reply:
Got it, thanks!